src/mame/rm/rm380z.h

```diff
@@ -13,6 +13,7 @@ Research Machines RM 380Z
 #pragma once
 
 #include "cpu/z80/z80.h"
+#include "emupal.h"
 #include "imagedev/cassette.h"
```

Comment:
Headers are grouped by library – headers from src/emu go after headers from src/devices (most dependencies to least dependencies, to make it less likely to end up with headers that need to be included in a specific magic order).

Do it like:
```c++
#include "cpu/z80/z80.h"
#include "imagedev/cassette.h"
#include "imagedev/floppy.h"
#include "machine/keyboard.h"
#include "machine/ram.h"
#include "machine/wd_fdc.h"

#include "emupal.h"
```

Reply:
Thanks, I've done this now.